opm/grid/cpgrid/DataHandleWrappers.cpp

```diff
@@ -1,3 +1,18 @@
 #include "DataHandleWrappers.hpp"
 
+#include <iostream>
+
 bool Dune::cpgrid::PointViaCellWarner::printWarn = true;
+
+void Dune::cpgrid::PointViaCellWarner::warn()
+{
+    if (printWarn)
+    {
+        std::cerr << "Communication of variable data attached to points is "
+                  << "not fully supported. Your code/handle must not use the"
```

Comment:
Maybe add a trailing blank here to avoid the message being `... use thelast parameter ...`?

Reply:
fixed.